### ISSUE TITLE
Abandon

### ISSUE THREAD

**Issue by digitaljohn** [julienpecorino/goportfolio#6]
Abandon this... old hat.

**Comment by julienpecorino** [comment]
Yep no worries and thanks a lot for all the help on the code !!!!

*Julien Pecorino*
Lead UX & UI Designer
—
UI Portfolio:   julien-pecorino.com
mail:        julien.pecorino@gmail.com
skype:     julien.pecorino
mobile:    0033 675 790 038
—
